lib/lookup.json

```diff
@@ -492,5 +492,10 @@
     "maintainers": ["mafintosh", "addaleax"],
     "prefix": "v",
     "skip": "<8"
+  },
+  "node-core-utils": {
+    "prefix": "v",
+    "maintainers": ["joyeecheung", "refack", "tiriel"],
```

Comment:
This can probably just be `"nodejs/automation"` right?

Reply:
Probably yes, I didn't know if it was possible to tag a team. Correcting right away.

Reply:
This field is just for CitGM members to know who to ping when things go wrong, so anything that can be @mentioned is fine.